editor/js/Sidebar.Project.js

```diff
@@ -61,7 +61,7 @@ function SidebarProject( editor ) {
 
 	//
 
-	/* container.add( new SidebarProjectMaterials( editor ) ); */
```

Comment:
Notice that the materials panel was commented out on purpose since the implementation was not satisfying, see e5139eb6d699c7e17dab669771007eb971d6a0a4. I don't think it's okay to add it back since nothing has changed since the removal.

Reply:
Neither e5139eb nor r126 gives a reason for the removal?   _"Editor: Commented out materials panel for now."_ 
 
It looks like @sdinesh86 asked the same question in #21369  when he incorporated the Materials pane in his PR.

Can you give any guidance on what makes it not satisfying?   

The only things extra I would prefer would be a "New Material" button.

Reply:
The general usability of the materials browsers wasn't good. 

The current implementation of the editor is not intended that materials, textures or geometries are created separately and then used to composed 3D objects like meshes. From my point of view, this implementing this requires a deeper refactoring of the editor's UI. 

I would also expect that materials can be previewed in a small window (similar to Blender) which makes it easier to debug and configure them.